pytest_rich.py

```diff
@@ -243,8 +244,48 @@ def pytest_sessionfinish(
             self.console.print(Rule("FAILURES", style="red"))
             for nodeid, report in self.failed_reports.items():
                 tb = RichExceptionChainRepr(nodeid, report.longrepr)
+                error_messages[nodeid] = tb.error_messages
                 self.console.print(tb)
 
+        if "-q" not in sys.argv:
+            self.print_summary(error_messages, all="-v" in sys.argv)
+
+    def print_summary(self, error_messages, all=False):
+        total_text = Text(f" Total::{self.total_items_completed}", style="bold blue")
+        success_text = Text(
+            f" Success::{self.total_items_completed - len(self.failed_reports)}",
+            style="bold green",
+        )
+        failed_text = Text(f" Failed::{len(self.failed_reports)}", style="bold red")
+        self.console.print(
+            Rule(
+                Text("Short test summary info")
+                + total_text
+                + success_text
+                + failed_text,
+                characters="=",
+                style="blue",
+            )
+        )
+
+        if all:
+            for nodeid, status in self.status_per_item.items():
+                if status == "success":
+                    self.console.print(
+                        Text("SUCCESS ", style="green")
+                        + Text(f"{nodeid}", style="black")
+                    )
+
+            self.console.print(Rule(characters="-", style="blue"))
+
+        for nodeid, errors in error_messages.items():
+            self.console.print(
+                Text("FAILED ", style="red")
+                + Text(f"{nodeid} {''.join(errors)}", style="black")
+            )
+
+        self.console.print(Rule("The End", characters="=", style="blue"))
```

Comment:
This is of course a matter of taste, but I don't think we need this rule at the end of the session... if we do want to have it however, we might display the exit status instead (given by `pytest_sessionfinish`).